metadata.go

```diff
@@ -216,13 +253,25 @@ func compileMetadata(
 	keyspace *KeyspaceMetadata,
 	tables []TableMetadata,
 	columns []ColumnMetadata,
+	functions []FunctionMetadata,
+	aggregates []AggregateMetadata,
 ) {
 	keyspace.Tables = make(map[string]*TableMetadata)
 	for i := range tables {
 		tables[i].Columns = make(map[string]*ColumnMetadata)
 
 		keyspace.Tables[tables[i].Name] = &tables[i]
 	}
+	keyspace.Functions = make(map[string]*FunctionMetadata, len(functions))
+	for i := range functions {
```

Comment:
Pretty please :-) ?

Just let's make it consistent with 
```golang
for _, function := range functions {
        keyspace.Functions[function.Name] = &function
}
```

Reply:
For some reason the tests fail with this change.

Reply:
yep, that's the reason.